blog/2025/03-01-ios-1.6/index.mdx

```diff
@@ -0,0 +1,89 @@
+- Fixed an issue that caused the app to display a blank screen sometimes when left open in the background.
+- Added user device name entitlement so the app can report the correct device name on iOS 16+.
+- Excluded unused features from the build so the app no longer requests access to permissions it doesn't need (like fitness data).
+
+### What is next?
+
+While the primary focus of the next couple releases will be to doing some dependency maintenance and updates, you can also expect work to
+continue on some exciting new features. Few people are probably aware that I started working on offline support
+[3 years ago](https://github.com/jellyfin/jellyfin-expo/pull/366)! Unfortunately it
+[wasn't quite ready](https://github.com/jellyfin/jellyfin-expo/issues/372) for this release, but my goal is to make it available as a
+general alpha/beta feature later this year. I have some other ideas to address some well-known limitations of the app and a new contributor
+has expressed interest in developing some new features also. You can track all new developments on the
+[project roadmap](https://github.com/orgs/jellyfin/projects/40/views/4) on GitHub.
+
+### Supported iOS versions
+
+The Jellyfin app currently supports iOS versions as old as iOS 12. Over the next few releases, we will be updating our core dependencies
+which will force us to increase our minimum supported version to iOS 15.1. If you are using a device that cannot be updated to a newer
+iOS version, you will still be able to use the Jellyfin web interface but future app updates will be unavailable.
+
+## Contributors
+
+Jellyfin is completely developed by volunteers, and couldn't be made without their great skills and dedication. Consider donating if you
+appreciate their work. A big shout-out to all contributors that made this release possible:
+
+**Jellyfin Team**
+
+- [@thornbill](https://github.com/thornbill) - Donate via [GitHub sponsors](https://github.com/sponsors/thornbill)
+- [@anthonylavado](https://github.com/anthonylavado) - Donate via [GitHub sponsors](https://github.com/sponsors/anthonylavado)
+- [@Bond-009](https://github.com/Bond-009) - Donate via [GitHub sponsors](https://github.com/sponsors/Bond-009)
+- [@JPKribs](https://github.com/JPKribs)
+
+**Other contributors**
+
+- [@Drew-Daniels](https://github.com/Drew-Daniels)
+- [@fidoriel](https://github.com/fidoriel)
+- [@Cyberbeni](https://github.com/Cyberbeni)
+
+And finally a big thank you to everyone who contributed translations, reported bugs, provided feedback and participated in beta testing!
+
+### Helping out
+
+If you have experience with React Native development and are interested in contributing yourself, feel free to dive into the
+[source code](https://github.com/jellyfin/jellyfin-expo) and open a pull request. Alternatively, you can help with translating the app
+into your own language on our [Weblate](https://translate.jellyfin.org/engage/jellyfin-expo/) instance.
```

Comment:
Can probably also mention contributing to jellyfin-web here, as those changes also end up on the iOS app.